src/Action/ExceptionAction.php

```diff
@@ -71,7 +74,13 @@ public function __invoke(FlattenException $exception, Request $request): Respons
         $headers['X-Content-Type-Options'] = 'nosniff';
         $headers['X-Frame-Options'] = 'deny';
 
-        return new Response($this->serializer->serialize($exception, $format['key'], ['statusCode' => $statusCode]), $statusCode, $headers);
+        $error = match ($format['key']) {
+            'jsonproblem' => $exception instanceof FlattenException ? $exception : new ProblemError($exception, $request->getPathInfo()),
```

Comment:
why the `$request->getPathInfo()` ? looks like a security issue

Reply:
You're most probably right, I wasn't too sure of how to hydrate the 'instance' part of the problem+json response, this is just a "filler" part while waiting for a global opinion on this

Reply:
Can you show me an example, I'm not sure I understand.